notebooks/examples/configure_check_conditions.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>The following guide includes different options for configuring a check's condition(s):</p><ul><li>Add Condition</li><li>etc...</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I meant to replace it (and delete the first sentence)</p><p> </p><p>If you leave it then I think it should be fixed to something like:</p><p> </p><p>Conditions are logic which runs on the check result's value and rate it as passing or failing. </p>